### PR TITLE
Redirect pages for old users documentation page locations

### LINKED ISSUE #420: Add redirects from the old documentation sections
Create HTML pages in each (top) folder of the old documentation structure, that will redirect to the new documentation pages. These do not show up in the documentation left-side navigation bar, and we can keep them (at least for a while) as a safety net.

### PR DESCRIPTION
**Pull request by thebe14**
# Summary

Links to user documentation in older documents/material are now incorrect, after the docs structure change. Redirect to the current page from old locations.

---

Fixes #420

### CODE REVIEW

**Comment by gwarf** [comment]
Thanks looks good.
Apparently you didn't create the PR from an up to date main branch, but once we squash the PR it should be fine, only the new files are changed.
The linter reports the missing `<title>` in the `<head>` elements, could you add them?

**Comment by gwarf** [comment]
I'm not sure about the title:     `<title>404 Page not found</title>`: fist the page will return an HTTP code 200 and it's more a redirection than a page not found.
Maybe you could replace by something more generic like `<title>Redirecting to new location</title>` ?

File: content/en/users/check-in/index.html
Comment (gwarf) [inline content/en/users/check-in/index.html:5]: Ideally you shouldn't use an hardcoded path like here, as it will fail on things like the test instance.

File: content/en/users/check-in/index.html
Comment (gwarf) [inline content/en/users/check-in/index.html:10]: It would be preferable to use an included image than relying on one in the wodpress site (that will get removed soon).

Comment (gwarf) [reply content/en/users/check-in/index.html:10]: We should already have a logo in the public dir.

**Comment by gwarf** [comment]
Have you bee looking at some point at https://gohugo.io/content-management/urls/#aliases?
I've also seen things like https://www.scripteverything.com/hugo-redirect-301-layout/.

But I'm totally fine with merging this PR for those pages to address this as quickly as possible, it's just that it could be interesting as an alternate way later.

File: content/en/users/check-in/index.html
Comment (gwarf) [reply content/en/users/check-in/index.html:5]: (this is something not ideal but I would be OK for merging despite this)

File: content/en/users/check-in/index.html
Comment (gwarf) [reply content/en/users/check-in/index.html:10]: I think it's the main point to be fixed before being able to merge.

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, looks great.
I've added a few comments, please review them. I would personally be OK for merging after having updated the title and used a local logo file.

**Comment by thebe14** [comment]
> Have you bee looking at some point at https://gohugo.io/content-management/urls/#aliases? I've also seen things like https://www.scripteverything.com/hugo-redirect-301-layout/.

Yes, it does not add anything. More complicated to setup and maintain. Also, these are temporary and we'll get rid of them in a relatively short period.

File: content/en/users/check-in/index.html
Comment (thebe14) [reply content/en/users/check-in/index.html:10]: The docu landing page also links to that. Will include it locally.

**Review by enolfc — APPROVED** [review]
LGTM if there are no broken links found :) Agree with @gwarf comments, but also need to merge asap

**Comment by thebe14** [comment]
> I'm not sure about the title: `<title>404 Page not found</title>`: fist the page will return an HTTP code 200 and it's more a redirection than a page not found. Maybe you could replace by something more generic like `<title>Redirecting to new location</title>` ?

Replacing titles....

**Comment by thebe14** [comment]
> Thanks looks good. Apparently you didn't create the PR from an up to date main branch, but once we squash the PR it should be fine, only the new files are changed. The linter reports the missing `<title>` in the `<head>` elements, could you add them?

But I did! First I did a rebase of my main branch from the EGI main one, took me half a day to merge the damn thing (also why I didn't push a PR on Friday). Then created a new branch from that, added the content I prepared Friday, and created the PR.

**Comment by thebe14** [comment]
Do not merge yet, let me replace with aliases. They seem to work okay.

File: content/en/users/check-in/index.html
Comment (gwarf) [reply content/en/users/check-in/index.html]: I was meaning the deployed preview, the PR is prepended in the URL path, see https://docs.egi.eu/documentation/423/users/check-in. Thus any `/something` will break.
If there is no simple solution it's acceptable for such a page.

**Comment by thebe14** [comment]
@enolfc @gwarf The remaining dead links are GitHub links (that are not publicly accessible?)

**Comment by gwarf** [comment]
> @enolfc @gwarf The remaining dead links are GitHub links (that are not publicly accessible?)

No, what I currently see is an error reaching an ops portal URL: https://github.com/EGI-Federation/documentation/runs/5460594664?check_suite_focus=true#step:4:228

**Review by gwarf — APPROVED** [review]

**Pull request merged**
